src/main/java/io/vertx/ext/prometheus/VertxPrometheusOptions.java

```diff
@@ -168,6 +168,16 @@ public boolean isEmbeddedServerEnabled() {
     return this;
   }
 
+  /**
+   * Alias for {@code enableEmbeededServer}
```

Comment:
Please, don't create an alias. Use the existing method or replace it with the new one.